optuna/samplers/_cmaes.py

```diff
@@ -40,6 +42,12 @@
 CmaClass = Union[CMA, SepCMA, CMAwM]
 
 
+class _CmaEsAttrKeys(NamedTuple):
+    optimizer: str
```

Comment:
The following code was prepared for users who used `CmaEsSampler` in prior to Optuna v2.5, but it seems old enough, so what do you think about removing it in this PR?

https://github.com/optuna/optuna/blob/8bef4de77284965a805560030f169eaa7efdcaf9/optuna/samplers/_cmaes.py#L490-L496

Reply:
OK. Let me remove it in the following pull request.